### PR TITLE
User add

### PR DESCRIPTION
**Pull request by srkirve77**
Added add user functionality to application

### CODE REVIEW

File: src/server-conn/postUser.jsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    {
+        method:'POST',
+        headers : {
+        'Content-Type': 'application/json' ,
+        'Authorization':'Bearer 9d8224224d0da981dc6768d900f96f7e7f5987d55971546ca180338c227e6c95'    
+        },
+        body: JSON.stringify(user),
+    })
+    const data = await res.json()
+    if(parseInt(data.code/100)!==2) {
```
Comment (Santosh-Punase) [inline src/server-conn/postUser.jsx]: Pls read about htttp `status` code

File: src/add-user/adduser.jsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    };
+
+    const handleCloseButtonFromModal = (CloseButton) => {
+        setOpen(CloseButton);
+    }
+    
+    return (
+        <div>
+        <Button variant="contained" color = "primary" className = {classes.root} onClick={handleOpen}>
+            <AddIcon className = {classes.addbutton} />Add User
```
Comment (Santosh-Punase) [inline src/add-user/adduser.jsx:42]: you can write test for button click

**Review by Santosh-Punase — CHANGES_REQUESTED** [review]

File: src/server-conn/postUser.jsx (hunk 1 above)
Comment (Santosh-Punase) [reply src/server-conn/postUser.jsx]: hint: don't manipulate it

File: src/add-user/adduser.jsx (hunk 2 above)
Comment (srkirve77) [reply src/add-user/adduser.jsx:42]: Okay

File: src/server-conn/postUser.jsx (hunk 1 above)
Comment (srkirve77) [reply src/server-conn/postUser.jsx]: Okay will do it some other way

**Review by Santosh-Punase — APPROVED** [review]
Comments are not resolved as expected. we will discuss this in a doubt session.
Approving for now so that you can proceed.

**Pull request merged**